src/retryable.rs

```diff
@@ -0,0 +1,190 @@
+use std::time::{Duration, SystemTime};
```

Comment:
This module needs a module-level documentation,
i.e stuff with the `//!` comments